lib/rows.js

```diff
@@ -264,8 +264,11 @@ function queryRowsOperationImpl(self, builtPlan, streamType, options, bindingArg
   );
   operation.validStatusCodes  = [200, 404];
   operation.complexValues  = complexValues;
+  if(graphqlQuery){
```

Comment:
I think spaces are needed here too - it looks like most of the code in this module has standard spaces before and after the parents. Is there a linter for node-client-api that would catch things like this?